### PR TITLE
U/fe/remove ncsa

### PR DESCRIPTION
**Pull request by frossie**
Remove LDF pages. This uncovered a bunch of broken references in the new USDF pages, which are now also fixed.

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -18,7 +18,7 @@ Check out the `README <https://github.com/lsst-dm/dm_dev_guide/blob/main/README.
 
 **Development guides:** :ref:`Overview <part-guides>` · :ref:`C++ <part-cpp>` · :ref:`Python <part-python>` · :ref:`Pybind11 <part-pybind11>` · :ref:`JavaScript <part-javascript>` · :ref:`ReStructuredText <part-rst>` · :ref:`DM Stack <part-dm-stack>` · :ref:`Git <part-git>` · :ref:`Editors <part-editors>` · :ref:`Legal <part-legal>` · :ref:`User documentation style <part-user-doc-style-guide>`
 
-**Services:** :ref:`Overview <part-services>` · :ref:`Jenkins <part-jenkins>` · :ref:`LSST Data Facility <part-ldf>`
+**Services:** :ref:`Overview <part-services>` · :ref:`Jenkins <part-jenkins>`
```
Comment (ktlim) [inline index.rst:21]: Maybe include IDF and USDF here, either under Services or on their own line?

File: services/data_protection.rst [hunk 2]
```diff
@@ -1,130 +0,0 @@
-########################################
```
Comment (ktlim) [inline services/data_protection.rst:1]: We should create a page like this for USDF.

File: services/monitoring_applications.rst [hunk 3]
```diff
@@ -1,59 +0,0 @@
-#################################################
```
Comment (ktlim) [inline services/monitoring_applications.rst:1]: We should create a page like this for USDF.

**Review by ktlim — APPROVED** [review]

File: services/data_protection.rst (hunk 2 above)
Comment (ktlim) [reply services/data_protection.rst:1]: (Not for Frossie to do; I have mentioned it to the USDF leads.)

File: services/monitoring_applications.rst (hunk 3 above)
Comment (ktlim) [reply services/monitoring_applications.rst:1]: (Not for Frossie to do; I have mentioned it to the USDF leads.)

**Review by womullan — APPROVED** [review]
OK for me - KTs comments I guess are a new action on USDF.

**Pull request merged**